src/ipscanner.py

```diff
@@ -0,0 +1,80 @@
+import socket
+from datetime import datetime
+import json
+import os
+from multi.scanner_thread import split_processing
+
+# Ask for input
+net1 = input('Enter the IP address: ')
+net2 = net1.split('.')
+a = '.'
+net3 = net2[0] + a + net2[1] + a + net2[2] + a
+
+# Print a nice banner with information on which host we are about to scan
+print("-" * 60)
+print("Please wait, scanning IP address....", net3+"XXX")
+print("-" * 60)
+
+# Resolves the relative path to absolute path
+# [BUG]: https://github.com/vinitshahdeo/PortScanner/issues/19
+
+
+def get_absolute_path(relative_path):
+    dir = os.path.dirname(os.path.abspath(__file__))
+    split_path = relative_path.split("/")
+    absolute_path = os.path.join(dir, *split_path)
+    return absolute_path
+
+
+# Check what time the scan started
+td1 = datetime.now()
+
+try:
+    with open(get_absolute_path('../config.json')) as config_file:
+        config = json.load(config_file)
+        # print get_absolute_path('../config.json')
+    range_low = int(config['ipRange']['low'])
```

Comment:
For `ipScanner.py` to read values *ipRange.low* and *ipRange.high* from config.json, these values should actually be present inside the config.json file.
You will have to update the *config.json* in `feature/python3.8`. Consider copying the *config.json* file from `master` branch.